upgrade/2022/r1-2022-sp1.md

```diff
@@ -0,0 +1,107 @@
+  ---
+title: R1 2022 SP1
+page_title: R1 2022 SP1 | for Telerik Reporting Documentation
```

Comment:
Remove " | for Telerik Reporting Documentation". Desi removed this for all our articles yesterday